classes/observers/listeners/MailDiscussionMessage.inc.php

```diff
@@ -48,20 +49,21 @@ public function handle(DiscussionMessageSent $event)
             $mailable = new \PKP\mail\mailables\MailDiscussionMessage($event->context, $submission);
             $emailTemplate = $mailable->getTemplate($event->context->getId());
 
-            $mailable->addData(array_merge(
-                [
-                    'siteTitle' => $mailable->viewData['journalName'],
-                ],
-                $event->formEmailData->getVariables($user->getId())
-            ));
-
             $mailable
                 ->body($emailTemplate->getLocalizedData('body'))
                 ->subject($emailTemplate->getLocalizedData('subject'))
                 ->sender($sender)
                 ->recipients([$user])
                 ->replyTo($event->context->getContactEmail(), $event->context->getContactName());
 
+            $mailable->setData(PKPLocale::getLocale());
```

Comment:
Is this necessary? The `Mail::send($mailable)` function should set the data before it is sent. And I think `siteTitle` should probably be set up in the `MailDiscussionMessage` mailable itself, rather than here in the listener, right?

Reply:
Forgot to rename `siteTitle`